frontend/model/contracts/group.js

```diff
@@ -1234,10 +1233,10 @@ sbp('chelonia/defineContract', {
         }
       }),
       process ({ contractID, data }, { state }) {
-        sbp('gi.contracts/group/pushSideEffect', contractID,
-          ['gi.contracts/group/releaseDeletedChatRoom', state.chatRooms[data.chatRoomID].members, data.chatRoomID]
-        )
-        delete state.chatRooms[data.chatRoomID]
+        const { chatRoomID } = data
+        const members = { ...state.chatRooms[chatRoomID].members }
+        delete state.chatRooms[chatRoomID]
+        sbp('gi.contracts/group/pushSideEffect', contractID, ['gi.contracts/group/releaseDeletedChatRoom', chatRoomID, members])
```

Comment:
There is no need to make these changes, please revert.

Reply:
Please see the `'gi.contracts/group/releaseDeletedChatRoom'` function definition below.
```
'gi.contracts/group/releaseDeletedChatRoom': (contractID, members) => {
  const identityContractID = sbp('state/vuex/state').loggedIn?.identityContractID
  if (identityContractID && members[identityContractID]?.status === PROFILE_STATUS.ACTIVE) {
    sbp('chelonia/contract/release', contractID).catch(e => {
      console.error('[gi.contracts/group/releaseDeletedChatRoom] Error', e)
    })
  }
}
```

The first parameter is contractID, and the second parameter is members. But the previous code was wrong in the order of passing params. First was `members`, and the second was `chatRoomID`. And since the `members` is reference to the `state.chatRooms[data.chatRoomID].members` and we delete the `state.chatRooms[data.chatRoomID]` just after calling `pushSideEffect`, it could be `undefined`. That's why I created another variable `members`.

Reply:
You are right that the order of the parameters needs to be fixed, but the rest of it can remain the same.

> And since the members is reference to the state.chatRooms[data.chatRoomID].members and we delete the state.chatRooms[data.chatRoomID] just after calling pushSideEffect, it could be undefined. That's why I created another variable members.

I'm pretty sure that isn't true and the only bug here was the order of the parameters.

Even though we do call `delete state.chatRooms[data.chatRoomID]`, that doesn't matter. the `members` will be saved as it is referenced and stored by the array that's passed to `pushSideEffect`. Try it and see.

Reply:
Okay.